src/server/graphql/queries/activeProOrgCount.js

```diff
@@ -0,0 +1,26 @@
+import {GraphQLInt} from 'graphql';
+import getRethink from 'server/database/rethinkDriver';
+import {requireSU} from 'server/utils/authorization';
+
+export default {
+  type: GraphQLInt,
+  async resolve(source, args, {authToken}) {
+    const r = getRethink();
+
+    // AUTH
+    requireSU(authToken);
+
+    // RESOLUTION
+    return r.table('Organization')
+      .filter((org) => org('tier').eq('pro'))
+      .map((org) => org('orgUsers')
```

Comment:
stripe enforces at least 1 seat per customer, so there's no way a 1-person org could have an inactive person.
we also never designed this capability into our own app, because there's no way you can pause yourself, so that means you'd have to simply not log in to go inactive. come to think of it, if that happened, i'm not sure what would happen in our app! I think stripe would just automatically turn the 0 seats into 1 seat.

all this to say, ` r.table('Organization').filter({tier: 'pro'}).count()` will suffice.

Reply:
ok!